test/integration/fixtures/profile/profiles.json

```diff
@@ -12,7 +12,7 @@
           "signatureRepresentation": 0,
           "keyType": "ECDSASecp256k1DER",
           "format": "jwt",
-          "didMethod": "orb",
```

Comment:
Looks like we have all converted to locally resolvable DIDs ? Do we have any example remaining for remotely resolvable DIDs ?

Reply:
I've revised the PR - kept in Orb tests.